### ISSUE TITLE
time interval "Crassicostatus"?

### ISSUE THREAD

**Issue by vjpsyverson** [paleobiodb/pbdb-new#24]
{"oid":"int:499","nam":"Crassicostatus","eag":171.60000,"lag":168.40000},
{"oid":"int:506","nam":"Crassicosta","eag":180.10000,"lag":175.60000},

Er...what? ,:-/

(data service call [here](https://training.paleobiodb.org/data1.2/combined/auto.json?show=countries&name=cra))

**Comment by mmcclenn** [comment]
Yes, those are actually in the table. Those are both in the timescale "Palfy North American Ammonite Chrons". I am working on redoing the timescale system now, but these intervals and others will certainly continue to be in the system.

**Comment by mmcclenn** [comment]
I will see if I can add a field to indicate what kind of interval it is, so we can add that to the autocomplete result display.

**Comment by vjpsyverson** [comment]
@mmcclenn, I know you've been working on timescale stuff lately -- did you do this yet?
